Homeworks/homework_04.py

```diff
@@ -1,16 +1,20 @@
+"""here is a docstring."""
```

Comment:
I can't run this code -> have an error during execution
Plus not all tasks are done. Seems code was not fully placed into pull reuest

Reply:
I'll close this PR and create another one. I will also add a link to the new PR in the LMS comments.